.gitguardian.yaml

```diff
@@ -0,0 +1,12 @@
+version: 2
```

Comment:
Remove `.gitguardian.yaml`. It's not needed now because I removed hardhat test private key from hardhat config and calculated it from seed.